test/spec/modules/userId_spec.js

```diff
@@ -644,9 +644,9 @@ describe('User ID', function () {
     it('pbjs.getUserIdsAsEids should prioritize user ids according to config available to core', () => {
       init(config);
       setSubmoduleRegistry([
-        createMockIdSubmodule('mockId1Module', {id: {uid2: {id: 'uid2_value'}}}),
+        createMockIdSubmodule('mockId1Module', {id: {uid2: {id: 'uid2_value'}, tdid: {id: 'uid1_value'}}}),
```

Comment:
here I try to address this comment: https://github.com/prebid/Prebid.js/pull/11119/files#r1496636311